feature-libs/organization/account-summary/occ/index.ts

```diff
@@ -0,0 +1 @@
+export * from './account-summary-occ.module';
```

Comment:
this `index.t`s can be removed. (It is better removed it now. When upgrading to Angular 14, if `index.ts` and `public_api.ts` both exist, we use `public_api.ts`, and remove `index.ts`. ng-packagr can automatically generate index)